sample_size/metrics.py

```diff
@@ -27,7 +27,7 @@ def variance(self) -> float:
     @staticmethod
     def check_positive(number: float, name: str) -> float:
         if number < 0:
-            raise Exception(f"Please provide a positive number for {name}.")
+            raise Exception(f"Error: Please provide a positive number for {name}.")
```

Comment:
Can we use a more specific Exception, like ValueError instead of the generic Exception